content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -105,6 +105,12 @@ must specify an IPv6 address, for example `--apiserver-advertise-address=fd00::1
 1. (Optional) Run `kubeadm config images pull` prior to `kubeadm init` to verify
 connectivity to the gcr.io container image registry.
 
+Swap must be disabled before initializing control-plane:
+
+```bash
+swapoff -a
```

Comment:
This is a one-off change I think? If the control plane host restarts, will that break things?